src/TwitchWindow.cpp

```diff
@@ -0,0 +1,61 @@
+#include "TwitchWindow.h"
```

Comment:
Please separate into a different PR.
Also, please avoid using the name "Twitch", it's a trademark.